synthtool/sources/templates.py

```diff
@@ -123,6 +124,13 @@ def language_pretty(input):
     return input
 
 
+#
+# converts Foo Bar into foo-bar, for use wih anchor links.
+#
+def slugify(input):
```

Comment:
This are both nits. :)

* Generally in Python function comments get placed right under the function definition.
* There are type hints for most of the functions, so let's continue to add them here.


```suggestion
def slugify(input: str) -> str:
"""Converts Foo Bar into foo-bar, for use wih anchor links."""
```

An example from this repo:
https://github.com/googleapis/synthtool/blob/0d263a3d31ba8844f166e483da87c26652e58085/synthtool/metadata.py#L43-L45

Reply:
Somewhat related - @busunkim96 are these things we can capture with a linter / syntax checker?  I like to make the robots do this kind of work for us in nodelandia 🙃